### PR TITLE
Migrate Mongo library integration to sandboxed api

### PR DESCRIPTION
**Pull request by labbati**
### Description

This PR adds and enable by default the sandboxed api for the mongo integration.
While developing this feature, I realize that for some reason we were not running the mongo test suite in CI, so I added both the required docker images and tests to CI too.

### Readiness checklist
- [x] (only for Members) Changelog has been added to the appropriate release draft. Create one if necessary.
- [x] Tests added for this feature/bug.

### Reviewer checklist
- [x] Appropriate labels assigned.
- [x] Milestone is set.
- [ ] Changelog has been added to the appropriate release draft. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: src/DDTrace/Integrations/IntegrationsLoader.php [hunk 1]
```diff
@@ -93,6 +94,8 @@ public function __construct(array $integrations)
                 '\DDTrace\Integrations\Memcached\MemcachedSandboxedIntegration';
             $this->integrations[MysqliSandboxedIntegration::NAME] =
                 '\DDTrace\Integrations\Mysqli\MysqliSandboxedIntegration';
+            $this->integrations[MongoSandboxedIntegration::NAME] =
+                '\DDTrace\Integrations\Mongo\MongoSandboxedIntegration';
```
Comment (morrisonlevi) [inline src/DDTrace/Integrations/IntegrationsLoader.php]: Nit: can you move this before Mysqli so it's in alphabetical order?

Comment (labbati) [reply src/DDTrace/Integrations/IntegrationsLoader.php]: Also, I should probably learn the alphabet :)

**Comment by labbati** [comment]
> This looks good to me, minus a nit. I don't know MongoDB, though. Does @SammyK have mongo experience? Maybe he could provide a review too since I'm not familiar with its API.

Please note that this is migrating legacy --> sandboxed api, is not about improving the mongo integration

File: composer.json [hunk 2]
```diff
@@ -125,6 +125,7 @@
             "@composer test -- tests/Integrations/Curl",
             "@composer test -- tests/Integrations/Memcached",
             "@composer test -- tests/Integrations/Mysqli",
+            "@composer test -- tests/Integrations/Mongo",
```
Comment (morrisonlevi) [inline composer.json]: Just noticed this one is out of alphabetical order too, but it's less important here.

**Pull request merged**